src/main/java/menion/android/whereyougo/maps/utils/LocusMapDataProvider.java

```diff
@@ -99,18 +98,16 @@ public void addZone(Zone z, boolean mark) {
 
         ArrayList<Location> locs = new ArrayList<>();
         for (int i = 0; i < z.points.length; i++) {
-            Location loc = new Location("");
-            loc.setLatitude(z.points[i].latitude);
-            loc.setLongitude(z.points[i].longitude);
+            Location loc = new Location(z.points[i].latitude,z.points[i].longitude);
             locs.add(loc);
         }
         if (locs.size() >= 3)
             locs.add(locs.get(0));
 
         Track track = new Track();
-        ExtraStyle style = new ExtraStyle();
-        style.setLineStyle(ColorStyle.SIMPLE, Color.MAGENTA, 2.0f, Units.PIXELS);
-        track.styleNormal = style;
+        GeoDataStyle style = new GeoDataStyle();
+        style.setLineStyle(Color.MAGENTA, 2.0f);
+        track.setStyleNormal(style);
```

Comment:
I'm not sure, if this is enough.
My solution would be
~~~
        LineStyle lineStyle = new LineStyle();
        lineStyle.setColoring(LineStyle.Coloring.SIMPLE);
        lineStyle.setColorBase(Color.MAGENTA);
        lineStyle.setWidth(2.0f);
        lineStyle.setUnits(LineStyle.Units.PIXELS);

        GeoDataStyle geoDataStyle = new GeoDataStyle();
        geoDataStyle.setLineStyle(lineStyle);
~~~

Reply:
Thanks for this. Used in 0a12e7b151d35b4c236698e4288a36214d6d8c21 and works.